OrbitService/ServiceUtilsTest.cpp

```diff
@@ -103,13 +103,15 @@ TEST(ServiceUtils, GetCumulativeTotalCpuTime) {
 
   const auto& jiffies1 = GetCumulativeTotalCpuTime();
```

Comment:
Maybe rename this one to `total_cpu_time1` as the type is now different?

Reply:
Good point. on it!

Reply:
done.